package.json

```diff
@@ -16,5 +16,8 @@
     "build": "parcel build ./client/index.html --public-url /static --out-dir static",
     "lint": "standard",
     "fix": "standard --fix"
+  },
+  "engines": {
+    "node": ">=14"
```

Comment:
Why such a very recent node version? If much I'd find 12 more appropriate 🤔

Reply:
Due to [this v8 post](https://v8.dev/features/optional-chaining), it should be 14+. In fact, we can remove the optional chaining operator by making the code compatible with a large number of versions. I only used it because I'm used to it and before there was no specified supported version of Node.